src/wmtk/simplex/SimplexCollection.cpp

```diff
@@ -103,4 +103,15 @@ SimplexCollection SimplexCollection::get_intersection(
     return sc;
 }
 
+bool SimplexCollection::are_simplex_collections_equal(
+    const SimplexCollection& collection_a,
+    const SimplexCollection& collection_b)
+{
+    if (collection_a.m_simplices.size() != collection_b.m_simplices.size()) {
+        return false;
+    }
+    SimplexCollection sc_union = SimplexCollection::get_union(collection_a, collection_b);
```

Comment:
I think you need to use intersection here instead of union. Otherwise `collection_a` could be a subset of `collection_b`.

Please also add a test for `are_simplex_collections_equal`.

Reply:
It cannot be. I already checked the length before